src/main/java/com/notably/view/MainWindow.java

```diff
@@ -79,6 +81,24 @@ public Stage getPrimaryStage() {
         return primaryStage;
     }
 
+    /**
+     * Provides an alternative way to set the settings of the main window, as an alternative
+     * to doing so via fxml tags.
+     *
+     * @param primaryStage the stage corresponding to the main app window.
+     */
+    private void setWindowSettings(Stage primaryStage) {
+        primaryStage.focusedProperty().addListener(((observable, unused, isFocused) -> {
+            if (isFocused) {
+                mainWindow.setEffect(null);
+            } else {
+                ColorAdjust colorAdjust = new ColorAdjust();
+                colorAdjust.setBrightness(-0.4);
+                mainWindow.setEffect(colorAdjust);
+            }
+        }));
+    }
+
```

Comment:
A consequence of this listener is that the whole Notably window got darkened when other apps are focused. What do you think about this? Should we just leave it as is for now?

Reply:
Oh, I see. Could you attach a screenshot of this behavior? I have not noticed this on my mac!